Riot/Modules/Room/DataSources/RoomDataSource.m

```diff
@@ -794,6 +794,7 @@ - (void)refreshFirstCellWithBeaconInfoSummary:(id<MXBeaconInfoSummaryProtocol>)b
     
     if (cellIndex != NSNotFound)
     {
+        [self updateCurrentUserLocationSharingStatus];
```

Comment:
It's safer to put it directly in the beacon info summary listener closure as it's possible to not have the bubbleData for current user in the bubbles list